.dagger/docs.go

```diff
@@ -198,6 +199,89 @@ func (d Docs) GenerateSchemaReference() *dagger.Directory {
 	return dag.Directory().WithFile(generatedAPIReferencePath, generatedHTML)
 }
 
+func (d Docs) GenerateReferencePages(ctx context.Context) (*dagger.File, error) {
```

Comment:
If we call this function in `Docs.Generate` then it'll automatically be updated when we make changes (and it'll also be caught in linting if it's out of date).

Reply:
Added to the function and updated the return type to `*dagger.Directory` to match.